app/routes/account.rb

```diff
@@ -34,6 +34,30 @@ class Account < Core
         current_user.reset_key
         redirect "/account"
       end
+
+      post '/account/track-mentor/invite' do
```

Comment:
I think it belongs in the account's routes.

The thing you're missing is in `app.rb` -- you'll see it if you scroll down.